dev/aarch64_opt/src/Makefile

```diff
@@ -0,0 +1,78 @@
+# SPDX-License-Identifier: Apache-2.0
+
+######
+# To run
+#
+# NOT IN THE NIX ENV
```

Comment:
Please remove this section or refer to the SLOTHY setup instructions. Repeating the SLOTHY setup here is only prone to get out of date.

Reply:
Apologies - will remove.

Reply:
Done